ui/src/Components/SearchCandidates.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { Table, Button, Col, Form, FormGroup, Label, Input } from 'reactstrap';
+import { SearchBox } from './search-box/search-box.component';
+import NavBar from './NavBar';
+
+export default class SearchCandidates extends React.Component {
+	constructor(props){
+		super(props);
+
+		this.state = {
+			loading: true,
+			search: null,
+			props: props,
+			searchName: null,
+			searchEmail: null,
+			searchNumber: null,
+		};
+	}
+
+
+	// async componentDidMount()
+	// {
+	// 	const url = `https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/search`;
+	// 	const resp = await fetch(url);
+	// 	const data = await resp.json();
+	// 	this.setState({search: data, loading:false})
+	// }
+	componentDidMount() {
+		fetch('https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/search')
+		  .then(response => response.json())
+		  .then(data => this.setState({ search: data, loading: false }));
+	}
+
+	/** Event Handlers for search fields
+		// TODO: Uncomment so state is updated, triggering a rerender of results
+	onNameSearch = (event) => {
+		this.setState({ searchName: event.target.value });
+		};
+
+	onEmailSearch = (event) => {
+		this.setState({ searchEmail: event.target.value });
+		};
+
+	onNumberSearch = (event) => {
+		this.setState({ searchNumber: event.target.value });
+		};
+
+	*/
+
+	render(){
+		console.log("RENDERING");
+		// Add results row by row
+		if (this.state.search) {
+			const obj = this.state.search.result; // instance of cleaner data queried earlier
+			var objLen = Object.keys(obj).length;
+			var items = [];
+
+			// Builds table with JSON object items passed in
+			items = []; // rebuild
```

Comment:
I'm not familiar with what this does, so just flagging it as it isn't something I have seen (declaring the variable items, then setting it to an empty array again). If this is intentional, then ignore :)

Reply:
Between the two items = [] there used to be a function declaration but now this doesn't make sense, I'll remove the second items declaration!